parts/k8s/kubernetescustomscript.sh

```diff
@@ -24,6 +24,12 @@ config_script=/opt/azure/containers/provision_configs.sh
 wait_for_file 3600 1 $config_script || exit $ERR_FILE_WATCH_TIMEOUT
 source $config_script
 
+if [[ "${TARGET_ENVIRONMENT,,}" == "azurestackcloud"  ]]; then 
```

Comment:
Let's add an `AZURE_STACK_ENV="azurestackcloud" # Azure Stack target env string` const-like thing to the `kubernetesprovisionsource.sh` file (we source it in above in the `source $script_lib` statement).

And then refer to `$AZURE_STACK_ENV` here whenever we want to compare a string to `"azurestackcloud"`.

Reply:
Fixed.